src/autolabel/transforms/base.py

```diff
@@ -96,18 +96,16 @@ async def apply(self, row: Dict[str, Any]) -> Dict[str, Any]:
             output = await self._apply(row)
         except Exception as e:
             logger.error(f"Error applying transform {self.name()}. Exception: {str(e)}")
-            output = {
-                k: self.NULL_TRANSFORM_TOKEN
-                for k in self.output_columns.values()
-                if k is not None
-            }
-            output[self.transform_error_column] = str(e)
+            output = {k: str(e) for k in self.output_columns.values() if k is not None}
```

Comment:
yeah it was an intentional change that we decided it would be better to send the error message to the LLM -> specifically, users can prompt the LLM to response differently in cases where a specific transform error occurs (returning a default value)